website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +86,60 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($('table.responsive-table-xxs thead').is(':hidden')) {
+        $('table.responsive-table-xxs tbody tr td:first-child.expanded').siblings().children().css('display', 'block');
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').off().on('click', function() {
```

Comment:
@sloria this is what I mentioned to you earlier.  With the exception of '.expanded' everything should be guaranteed to be there based on the definition of the plugin/what the plugin will create for you.  And some of the selectors are necessary to get the state of the DOM efficiently (`if ($('table.responsive-table-xxs thead').is(':hidden')` determines if the media query defined in contributor-page.css is active or not, in other words has the table been collapsed or not).